package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "bitflux-openfin",
   "version": "10.0.0-rc.1",
+  "type": "develop",
```

Comment:
Yes, to do a proper release you want to change that to master and yes, automating it in grunt is a great idea! Will get on it.